geanyvc/src/geanyvc.c

```diff
@@ -1676,7 +1677,7 @@ create_commitDialog(void)
 	gtk_widget_show(dialog_action_area1);
 	gtk_button_box_set_layout(GTK_BUTTON_BOX(dialog_action_area1), GTK_BUTTONBOX_END);
 
-	btnCancel = gtk_button_new_from_stock("gtk-cancel");
+	btnCancel = gtk_button_new_with_mnemonic(_("_Cancel"));
```

Comment:
This is gonna lose the potential icon on the button.  Not the end of the world though, but Geany itself has them if enabled at the system level.